src/state/store.js

```diff
@@ -71,8 +71,12 @@ function getDefaultState(
 
 export const store = createStore(getDefaultState({ basedOnQueryParams: true }));
 
-store.reset = () => {
+store.reset = (home = false) => {
   store.reinitialize();
+  if (home) {
+    // if user clicks on home page, reset state to not include query params
+    store.setState(getDefaultState());
+  }
```

Comment:
It looks like maybe the `setState` call overrides and doesn't need the `reinitialize` call above it if it's `home`. If that's true, can we skip that call and only `reinitialize` in an `else` clause?

Reply:
makes sense! i just updated the logic here in a new commit